tests/legacy/system/test_edb.py

```diff
@@ -1028,6 +1028,25 @@ def test_hfss_simulation_setup(self):
         assert mop.number_of_layer_elements == "3"
         edbapp.close()
 
+    def test_hfss_simulation_setup_b(self, edb_examples):
+        edbapp = edb_examples.get_si_verse()
+        setup1 = edbapp.create_hfss_setup("setup1")
+        """sweep1 = setup1.add_frequency_sweep(name="sweep1",
+                                   frequency_sweep=[
+                                   ["linear count", "1MHz", "10MHz", 10],
+                                   ])
+        sweep2 = setup1.add_frequency_sweep(name="sweep2",
+                                            frequency_sweep=[
+                                                ["log scale", "1kHz", "100kHz", 10],
+                                            ])"""
```

Comment:
```suggestion
        sweep1 = setup1.add_frequency_sweep(name="sweep1",
                                   frequency_sweep=[
                                   ["linear count", "1MHz", "10MHz", 10],
                                   ])
        sweep2 = setup1.add_frequency_sweep(name="sweep2",
                                            frequency_sweep=[
                                                ["log scale", "1kHz", "100kHz", 10],
                                            ])
```
Why is this part of the code "commented" ("""...""") ?
Also, could you add assert calls to see if everything worked fine ?